benchmark/README.md

```diff
@@ -0,0 +1,18 @@
+Benchmarking
+============
+
+
+
+To build all of the benchmarks:
+```
+# Run from project root directory
+mvn clean install
+```
+
+
+To run a benchmark jar, run the following command
+```
+# Run from project root directory
+ java -jar target/benchmark.jar
```

Comment:
this still doesn't work when I clone this repo and run it on this branch specifically.

Reply:
resolved -- needed to run from benchmark directory